src/components/layout/index.js

```diff
@@ -7,16 +7,18 @@ import Footer from "./Footer"
 
 import "./index.module.css"
 
-const Layout = ({ children }) => (
-  <div styleName="root">
-    <Helmet>
-      <link rel="stylesheet" href="https://use.typekit.net/bcx8qfd.css" />
-    </Helmet>
-    <Header />
-    <main>{children}</main>
-    <Footer />
-  </div>
-)
+const Layout = ({ children }) => {
```

Comment:
Why this change?

Reply:
Nevermind, these are leftovers from some experiences. I'll revert this